addons/binding/org.openhab.binding.valloxmv/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,34 @@
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: ValloxMV Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.valloxmv;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Import-Package: 
+ javax.measure,
+ javax.measure.quantity;version="1.0.0",
```

Comment:
Is this version number, needed ideally it should be left out.

Reply:
Changed